{
  "uuid": "981c1826-501c-4d30-a211-618fb21310f4",
  "name": "OdinUiTests.Tests.Activity.TaskWithSolutionStudentTests.Появление ошибки при удалении решения",
  "children": [
    "666a6376-cad7-40c7-afb0-f3b68c775e58"
  ],
  "befores": [],
  "afters": [],
  "links": [],
  "start": 1753145973755,
  "stop": 1753145988965
}